config/Config-kernel.in

```diff
@@ -1125,6 +1125,12 @@ if KERNEL_IP_MROUTE
 
 endif
 
+config KERNEL_MPTCP
+	bool "Enable IPv4 MultiPath TCP support"
```

Comment:
Oh, and you don't need to explicitly say IPv4 here.

Reply:
Users might be surprised when they then encounter ipv6 further in.

Reply:
Done! In addiction, i've defined that MultiPath is enabled by default (IPv4 and IPv6).